app/models/form_profiles/va_526ez.rb

```diff
@@ -130,14 +130,25 @@ def initialize_vets360_contact_info
   def initialize_veteran_contact_information
     return {} unless user.authorize :evss, :access?
 
-    # fill in blank values with PCIU data
-    return_val = initialize_vets360_contact_info.merge(
-      mailing_address: get_common_address,
-      email_address: extract_pciu_data(:pciu_email),
-      primary_phone: pciu_us_phone
-    ) { |_, old_val, new_val| old_val.presence || new_val }
-
-    contact_info = VA526ez::FormContactInformation.new(return_val)
+    contact_info = if Flipper.enabled?(:disability_compensation_remove_pciu)
+                     Rails.logger.info('disability_compensation_remove_pciu=TRUE')
+                     initialize_vets360_contact_info
+                   else
+                     # fill in blank values with PCIU data
+                     Rails.logger.info('disability_compensation_remove_pciu=FALSE')
+                     initialize_vets360_contact_info.merge(
+                       mailing_address: get_common_address,
+                       email_address: extract_pciu_data(:pciu_email),
+                       primary_phone: pciu_us_phone
+                     ) { |_, old_val, new_val| old_val.presence || new_val }
+                   end
+    # Logging was added above and below here to contrast/compare completeness of contact information returned
+    # from VA Profile alone versus VA Profile + PCIU. This logging will be removed when the Flipper flag is.
+    Rails.logger.info("mailing_address=#{contact_info[:mailing_address].present?}")
+    Rails.logger.info("email_address=#{contact_info[:email_address].present?}")
+    Rails.logger.info("primary_phone=#{contact_info[:primary_phone].present?}")
+
```

Comment:
Yeah, I was kind of wondering that as well, so I appreciate the input here. Combining them all in one line would look pretty gnarly from a readability perspective, but it might be wise to combine them anyway to minimize logging calls...